pkg/mkimage-raw-efi/make-raw

```diff
@@ -379,12 +381,21 @@ INITRD_IMG=/parts/initrd.img
 INSTALLER_IMG=/parts/installer.img
 # GRUB cfg override for our installer
 INSTALLER_GRUB_CFG=/parts/grub.cfg
+# IMX8 flash blob that needs write to image with 33k offset
+IMX8_BLOB=/parts/imx8-flash.bin
 
 # offset in the GPT partition table from which we can start our numbering
 PART_OFFSET=0
 # starting sector for our portion of the disk
 CUR_SEC="$FIRST_PART_SEC"
 
+if [ -f "$IMX8_BLOB" ]; then
+  # For imx8 boards we need write boot blob with 33kb offset.
+  IMX_BLOB_END_SEC=$(( IMX8_BLOB_SIZE / 512 ))
+  CUR_SEC="$IMX_BLOB_END_SEC"
+  dd if="$IMX8_BLOB" of="$IMGFILE" bs=1024 seek=33 conv=notrunc
```

Comment:
Other i.MX8M uses different offset than 33 (1 or 32), so I've created <platform>-flash.conf file to point to the corresponding offset:
https://github.com/rene/eve/commit/c2d7fdac341b99840313fef837216f7c12ddbb24#diff-ed9e279b2040ba569bb9874c24eb1e6d03962262cd3f4dbef55200e7bb65aa78L396